capstone/src/main/java/com/google/sps/Objects/Challenge.java

```diff
@@ -1,57 +1,128 @@
 package com.google.sps.Objects;
 
 import java.util.ArrayList;
+import java.util.List;
 
 import com.google.appengine.api.datastore.Entity;
 
+/**
+ * Represents the current challenge in a group.
+ *
+ * @author lucyqu
+ */
 public final class Challenge {
 
   private final String challengeName;
   private final long dueDate;
   private final Badge badge;
-  private final ArrayList<String> usersCompleted;
+  private final List<String> usersCompleted;
```

Comment:
I learned in school that it's better to initialize as List rather than ArrayList because it's more generic and noticed that here I left it as ArrayList